packages/middleware-retry/src/isStreamingPayload/isStreamingPayload.ts

```diff
@@ -0,0 +1,12 @@
+import type { HttpRequest } from "@smithy/protocol-http";
+import { Readable } from "stream";
+
+/**
+ * @internal
+ */
+export const isStreamingPayload = (request: HttpRequest): boolean => {
+  return (
+    request?.body instanceof Readable ||
+    (typeof ReadableStream !== "undefined" && request?.body instanceof ReadableStream)
+  );
+};
```

Comment:
nit: brackets can be removed
```suggestion
export const isStreamingPayload = (request: HttpRequest): boolean => 
  request?.body instanceof Readable ||
  (typeof ReadableStream !== "undefined" && request?.body instanceof ReadableStream);
```